File: app/src/main/java/com/utynote/components/search/model/geojson/entities/Engine.java

package com.utynote.components.search.model.geojson.entities;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Engine {

    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("author")
    @Expose
    public String author;
    @SerializedName("version")
    @Expose
    public String version;

}

File: app/src/main/java/com/utynote/components/search/data/geojson/entities/Engine.kt
package com.utynote.components.search.data.geojson.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Engine {

    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("author")
    @Expose
    var author: String? = null
    @SerializedName("version")
    @Expose
    var version: String? = null

}
